app/src/main/java/com/brewfind/dto/Brewery.kt

```diff
@@ -2,7 +2,11 @@ package com.brewfind.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class Brewery(@SerializedName("Name") var name: String, @SerializedName("brewery_type") var size: String, @SerializedName("street") var street: String) {
+data class Brewery(@SerializedName("name") var name: String,
+                   @SerializedName("brewery_type") var size: String,
+                   @SerializedName("street") var street: String,
+                   @SerializedName("city") var city: String,
+                   @SerializedName("state") var state: String){
```

Comment:
Added two variables and their API values, since in the README file it mentions searching for breweries by region which included state and city. I added those two since they were both mentioned in the API, and I felt that it was something that would be useful for the project's vision.